machines/Makefile.include

```diff
@@ -0,0 +1,11 @@
+include arch_filelist.mk
```

Comment:
do we still need this file? I don't see it being included.

Reply:
Which file? Arch_filelist.mk?

Reply:
no the "Makefile.include". is this supposed to be included in "machines/Makefile"?

Reply:
Oh, you’re right. It should be removed

Reply:
Fixed